specs/_features/eip7594/p2p-interface.md

```diff
@@ -49,20 +49,20 @@
 
 ```python
 class DataColumnSidecar(Container):
-    index: LineIndex  # Index of column in extended matrix
+    index: ColumnIndex  # Index of column in extended matrix
     column: DataColumn
     kzg_commitments: List[KZGCommitment, MAX_BLOB_COMMITMENTS_PER_BLOCK]
     kzg_proofs: List[KZGProof, MAX_BLOB_COMMITMENTS_PER_BLOCK]
     signed_block_header: SignedBeaconBlockHeader
     kzg_commitments_inclusion_proof: Vector[Bytes32, KZG_COMMITMENTS_INCLUSION_PROOF_DEPTH]
 ```
 
-#### `DataColumnIdentifier`
+#### `DataColumnIndexentifier`
 
 ```python
-class DataColumnIdentifier(Container):
+class DataColumnIndexentifier(Container):
```

Comment:
I assume a `sed` command broke down here, or I found my new favorite word.

Reply:
No, you found the worst word. 👍